journals/apps/journals/models.py

```diff
@@ -314,6 +317,35 @@ def generate_basket_url(self, sku):
         ))
         return basket_url
 
+    def update_related_objects(self, deactivate=False):
+        data = {
+            "status": "active" if not deactivate else "inactive",
+            "card_image_url": self.card_image_absolute_url,
+            "title": self.title,
```

Comment:
As the title is used in ecommerce (on the basket page), we need to update ecommerce as well when the title changes

Reply:
I'll write the code for it, but the current patch endpoint requires stockrecords, attribute_values, and product_class, which I won't have at the time to update it. I could query ecommerce, then write to ecommerce, but it would be nice if there was a way to just update the title.